.github/workflows/release.yml

```diff
@@ -3,7 +3,7 @@ name: Release
 on:
   push:
     tags:
-      - 'v*'
+      - 'v*.*.*'
```

Comment:
I don't think this is particularly strict. I'd rather leave as it was or go for something that is quite specific (e.g. `'v[0-9]+.[0-9]+.[0-9]+'`).